forwards_paper.tex

```diff
@@ -180,13 +180,12 @@ \section*{Results}
 
 We begin by benchmarking the performance improvement achieved by connecting
 two previously published forwards-time simulation libraries,
-\simupop{} \citep{simupop} and \fwdpp{} \citep{fwdpp}
-to the tools we describe here, implemented in \msprime{}, allowing them to
-efficiently record the population pedigree in a simplified tree sequence.
+\simupop{} \citep{peng2005simupop} and \fwdpp{} \citep{fwdpp}
+to the tools we describe here, as implemented in \msprime{}.
 Then, we describe the conceptual and algorithmic foundations for the method:
 (a) a format, implemented in the \msprime{} Python API,
 for recording tree sequences in several \emph{tables};
-(b) an algorithm, \algref{W}, to record tree sequence information into these tables on the fly
+(b) an algorithm to record tree sequence information into these tables on the fly
```

Comment:
I think it's clear below what algorithm it is, and forward-referring to it as Alg W seems confusing.

Reply:
+1